perftest/perftest.go

```diff
@@ -189,17 +196,13 @@ func (p *perfLoadTesting) depositTokens(assets map[string]string, opts Opts) err
 }
 
 func (p *perfLoadTesting) checkNetworkLimits(opts Opts) error {
-	// Check the limit of the number of orders per side in the LP shape
-	networkParam, err := p.dataNode.getNetworkParam("market.liquidityProvision.shapes.maxSize")
+	// Get the stake to liquidity scaling value
+	networkParam, err := p.dataNode.getNetworkParam("market.liquidityV2.stakeToCcyVolume")
```

Comment:
this parameter will be renamed in a soon PR, not a big deal here for now, just so you know it might break and that will be ther reason